qiskit/pulse/instructions/delay.py

```diff
@@ -58,7 +58,7 @@ def channel(self) -> Channel:
     @property
     def channels(self) -> Tuple[Channel]:
         """Returns channels that this schedule uses."""
-        return tuple((self.channel, ))
+        return self.channel,
```

Comment:
It's fine either way -- and I'm sorry to nitpick over this a _second_ time -- `(self.channel,)` seems to be the best option because it feels very explicit that a tuple is being returned, without redundantly calling `tuple` on a tuple. the `,` could be easy to miss or confusing.

But, I don't want to be a pain, it's fine if you want to leave it 😛

Reply:
Thanks! actually only `,` causes linter error. Now it is `(self.channel, )`